src/components/Avatar/Examples.stories.tsx

```diff
@@ -192,3 +193,63 @@ environmentModel.argTypes = {
   spotLightIntensity: { control: { type: 'range', min: 0, max: 20, step: 0.1 } },
   environment: { options: Object.keys(environmentPresets), control: { type: 'select' } }
 };
+// @ts-ignore
+export const ReflectiveFloor: StoryFn<typeof Avatar> = (
+  args: AvatarProps & FloorReflectionProps & { debug: boolean }
+) => (
+  <Avatar {...args} effects={{ ambientOcclusion: true }} style={{ background: args.color }}>
+    <FloorReflection {...args} />
+    {args?.debug && <StatsGl />}
+  </Avatar>
+);
+ReflectiveFloor.args = {
+  ...Static.args,
+  fov: 50,
+  // @ts-ignore
+  shadows: false,
+  modelSrc: modelPresets.one,
+  animationSrc: animationPresets.three,
+  environment: 'warehouse',
+  /* eslint-disable no-console */
+  onLoaded: () => console.info('EVENT: environment model loaded'),
+  /* eslint-enable no-console */
+  // @ts-ignore
+  resolution: 512,
+  mixBlur: 0.8,
+  mixStrength: 80,
+  metalness: 0.5,
+  blur: [300, 200],
+  mirror: 1,
+  minDepthThreshold: 0.4,
+  maxDepthThreshold: 1.4,
+  depthScale: 1.2,
+  depthToBlurRatioBias: 1,
+  distortion: 0,
+  mixContrast: 1,
+  reflectorOffset: 0,
+  roughness: 1,
+  color: 'rgb(9,20,26)',
+  debug: false
+};
+
+ReflectiveFloor.argTypes = {
+  ...ignoreArgTypesOnExamples,
+  // @ts-ignore
+  fov: { control: { type: 'range', min: 30, max: 100, step: 1 } },
+  // @ts-ignore
+  environment: { options: Object.keys(environmentPresets), control: { type: 'select' } },
+  // @ts-ignore
+  resolution: { control: { type: 'range', min: 64, max: 2048, step: 64 } },
+  mixBlur: { control: { type: 'range', min: 0, max: 10, step: 0.1 } },
+  mixStrength: { control: { type: 'range', min: 0, max: 100, step: 1 } },
+  metalness: { control: { type: 'range', min: 0, max: 1, step: 0.01 } },
+  mirror: { control: { type: 'range', min: 0, max: 10, step: 1 } },
```

Comment:
it's 0 or 1

    mirror={0} // Mirror environment, 0 = texture colors, 1 = pick up env colors

Reply:
Changed